webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/v2/searchResourceByLabelSubQuery.scala.txt

```diff
@@ -31,7 +31,7 @@
     SELECT (count(distinct ?resource) as ?count)
 }
 WHERE {
-    ?resource <http://jena.apache.org/text#query> "@searchTerm.generateLiteralForLuceneIndexWithoutExactSequence" .
+    ?resource <http://jena.apache.org/text#query> (rdfs:label "@searchTerm.generateLiteralForLuceneIndexWithoutExactSequence") .
```

Comment:
what influence does this change have? has our search by label been defective all along?

Reply:
The update doesn't affect anything. It's just good practice to be as explicit as possible, according to the [Apache Jena documentation](https://jena.apache.org/documentation/query/text-query.html), so I thought I add this.